src/archiefvernietigingscomponent/theme/models.py

```diff
@@ -0,0 +1,93 @@
+import re
+
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+
+color_re = re.compile(r"^[0-9a-fA-F]+")
+validate_color = RegexValidator(
+    color_re,
+    _("Enter a valid HTML color code value consisting of only hexidecimal characters."),
+    "invalid",
+)
+
+
+class ThemeConfig(SingletonModel):
+    logo = models.FileField(
+        upload_to="theme/",
+        help_text=_(
+            "Image will be scaled down to a height of 100 pixels. Typically upload a picture with a height of 200 pixels."
+        ),
+    )
+
+    organization_name = models.CharField(
+        _("Organization name"),
+        max_length=100,
+        help_text=_("Shown on the top of the footer."),
```

Comment:
Maybe better (so it's also consistent with the address helptext):
```suggestion
        help_text=_("Shown at the top of the footer."),
```